### PR TITLE
Takeover pages

### PR DESCRIPTION
**Pull request by nehasri89**
The feature is to allow Takeover pages to be served from Content API, the way to do it is it pretends to be a react page, but TC throws an exception for the this template type `takeoverPage` which the render catches and reverts back to the old rendering mechanism using content api and dust template. 

`takeoverPage`  a new template type introduced to identify takeover pages.

The corresponding Render PR https://github.com/newsuk/cps-content-render/pull/2811

### CODE REVIEW

File: packages/ssr/src/server/article.js [hunk 1]
```diff
@@ -62,5 +66,7 @@ module.exports = (
     name: "article"
   };
 
-  return runServer(article, options);
+  return runServer(article, options).catch(error => {
+    throw new Error(error);
```
Comment (parkjh) [inline packages/ssr/src/server/article.js]: @nehasri89 and is this suppose to be caught in render?

File: packages/article/src/article.js [hunk 2]
```diff
@@ -17,11 +17,22 @@ export const templates = {
   mainstandard: ArticleMainStandard
 };
 
+export class TakeoverBailout extends Error {
```
Comment (parkjh) [inline packages/article/src/article.js]: @nehasri89 do we not have a more concise error handling package?

File: packages/ssr/src/server/article.js (hunk 1 above)
Comment (nehasri89) [reply packages/ssr/src/server/article.js]: yup

File: packages/article/src/article.js (hunk 2 above)
Comment (nehasri89) [reply packages/article/src/article.js]: not at the moment

Comment (ValGeorgiev) [reply packages/article/src/article.js]: We should use ErrorBoundaries https://reactjs.org/docs/error-boundaries.html

Comment (parkjh) [reply packages/article/src/article.js]: @ValGeorgiev @nehasri89 maybe as a separate piece of work? but I would have assumed that error handling would've been one of the first thing to build as a package...

**Pull request merged**